resources/keywords/login.resource

```diff
@@ -88,3 +90,23 @@ User Closes Welcome Message
     Run Keyword If    ${welcomeTile}
     ...    Click Element    ${btnWelcomTileClose}
 
+The User Logs In To Openshift As ${user}
+    Set Selenium Implicit Wait    5s
+    Open Browser    ${OCP_CONSOLE_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Select Login Authentication Type    ${${user}.AUTH_TYPE}
+    Login To Openshift As ${user}
+    Login To OpenShift CLI As LDAP User     ${${user}.USERNAME}     ${${user}.PASSWORD}     ${DBaaSTestNS}
```

Comment:
Why login to CLI appears here? Aren't we dealing with UI in this case?

Reply:
We are using CLI as well as UI to perform operations like creating service binding which can be only performed with CLI.